systemtest/src/main/java/io/strimzi/systemtest/matchers/LogHasNoUnexpectedErrors.java

```diff
@@ -73,7 +73,8 @@ public void describeTo(Description description) {
         OPERATION_TIMEOUT("Util:[0-9]+ - Exceeded timeout of.*while waiting for.*"),
         // This is whitelisted cause it's no real problem when this error appears, components are being created even after timeout
         RECONCILIATION_TIMEOUT("ERROR Abstract.*Operator:[0-9]+ - Reconciliation.*"),
-        ASSEMBLY_OPERATOR_RECONCILIATION_TIMEOUT("ERROR .*AssemblyOperator:[0-9]+ - Reconciliation.*[fF]ailed.*");
+        ASSEMBLY_OPERATOR_RECONCILIATION_TIMEOUT("ERROR .*AssemblyOperator:[0-9]+ - Reconciliation.*[fF]ailed.*"),
+        WATCHER_CLOSED_EXCEPTION("ERROR AbstractOperator:* - Watcher closed with exception in namespace connect-cluster-test");
```

Comment:
Namespace should be generic here. I think it can occurred in all namespaces which we use.

Reply:
Yeah my bad, sorry